src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/Notifications/bit-message-bar.fluent.scss

```diff
@@ -1,99 +1,213 @@
 ﻿@import "../colors.fluent.scss", "../../functions.scss";
 
 .bit-msg-bar-fluent {
+    background-color: $Grey20;
+    color: $Grey160;
     font-family: FluentFont;
     font-size: rem(12px);
     font-weight: 400;
-    line-height: inherit;
-    color: $Grey160;
-    align-items: center;
+    min-height: 32px;
     width: 100%;
     display: flex;
     word-break: break-word;
-    min-height: 32px;
-    align-items: flex-start;
 
-    & > span:nth-child(2) {
-        flex-grow: 1;
-        margin: 8px 0px 8px 12px;
-        padding: 0;
-        text-align: left;
-        overflow: hidden;
-        white-space: nowrap;
-        text-overflow: ellipsis;
+    &.bit-msg-bar--error, &.bit-msg-bar--blocked {
+        background-color: $red20;
     }
 
-    & > .not-truncate:nth-child(2) {
-        overflow: visible;
-        white-space: normal;
+    &.bit-msg-bar--severeWarning {
+        background-color: $orange20;
     }
 
-    & > span:first-child {
-        display: inline-block;
-        margin: 8px 0 8px 12px;
+    &.bit-msg-bar--success {
+        background-color: $green60;
+    }
+
+    &.bit-msg-bar--warning {
+        background-color: $yellow20;
+    }
+
+    &.bit-msg-bar-multiline {
+        flex-direction: column;
+    }
+
+    .bit-msg-bar-content {
+        display: flex;
+        width: 100%;
+        line-height: normal;
+    }
+
+    .bit-msg-bar-icon {
         font-size: 16px;
+        min-width: 16px;
+        min-height: 16px;
+        display: flex;
+        flex-shrink: 0;
+        margin: 8px 0 8px 12px;
+    }
+
+    .bit-msg-bar-icon .bit-icon {
+        color: $ThemeLighter;
     }
 
-    & > span:last-child {
+    &.bit-msg-bar--error .bit-msg-bar-icon .bit-icon,
+    &.bit-msg-bar--blocked .bit-msg-bar-icon .bit-icon,
+    &.bit-msg-bar--severeWarning .bit-msg-bar-icon .bit-icon {
+        color: $red180;
+    }
+
+    &.bit-msg-bar--success .bit-msg-bar-icon .bit-icon {
+        color: $green120;
+    }
+
+    &.bit-msg-bar--warning .bit-msg-bar-icon .bit-icon {
+        color: $Grey160;
+    }
+
+    @media screen and (-ms-high-contrast: active) {
+        .bit-msg-bar-icon {
+            -ms-high-contrast-adjust: none;
+            color: WindowText;
+        }
+    }
+
+    .bit-msg-bar-text {
+        min-width: 0;
         display: flex;
+        flex-grow: 1;
+        margin: 8px;
+        font-family: FluentFont;
+        font-size: rem(12px);
+        font-weight: 400;
+    }
+
+    .bit-msg-bar-text:not(.bit-msg-bar-dismissSingleLine) {
```

Comment:
Please change the class namings conformed to our convention we only use kebab-case and avoid long names